kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/ServiceControlCliArgumentsSuite.scala

```diff
@@ -67,6 +67,9 @@ class ServiceControlCliArgumentsSuite extends KyuubiFunSuite {
       }
       thread.start()
       thread.join()
+      // scalastyle:off
+      println(logAppender.loggingEvents.toString())
+      // scalastyle:on
```

Comment:
is it an expect print ?

Reply:
Sorry, I added it when I updated the doc and ut, but I forgot to remove it.